docs/integrations/github.md

```diff
@@ -91,6 +102,7 @@ It has two boolean flags that can be passed in to enable additional functionalit
 * `--delete` - This will delete the PR environment after deploying to production. 
     * Note: If using `--delete` then the runner will need a connection to the engine even if you are using Airflow. This is because the delete is done outside of Airflow.
     * Eventually want the SQLMesh Janitor to automatically do this cleanup which would remove the need for this flag.
+* `--command_namespace` - Namespace to use for SQLMesh commands. For example if you provide `SQLMesh` as a value then commands will be expected in the format of `#SQLMesh/<command>`. The `#` prefix is automatically added.
```

Comment:
I'm a little confused by the 

> The `#` prefix is automatically added.

and 

> will be expected in the format of `#SQLMesh/<command>`

aren't these statements contradicting?

Reply:
I will just remove the pound sign and let the user set whatever namespace they want to avoid this confusion.